src/express.re

```diff
@@ -1,23 +1,27 @@
-let module Next = {
+module Next = {
   type t;
 };
 
-let module Request = {
+module Request = {
   type t;
 };
 
-let module Response = {
+module Response = {
   type t;
   external sendFile : t => string => 'a => unit = "" [@@bs.send];
```

Comment:
The last argument here looks like it could be either a `Js.t` or a function of some sort. If you don't want to break backwards-compatibility right now, you could perhaps add a note about what it _should_ be.